.github/workflows/auto-upgrade-certified-connectors-cdk.yml

```diff
@@ -158,4 +158,6 @@ jobs:
       # We've pushed all the changes to the PR. Mark it ready for review.
       - name: Ready for review
         if: steps.check-changes.outputs.has_changes == 'true'
-        run: gh pr ready "$pr_number"
+        run: gh pr ready "${{ steps.create-pr.outputs.pull-request-number }}"
+        env:
+          GH_TOKEN: ${{ steps.app-token.outputs.token }}
```

Comment:
no idea how this was working previously, but I ran into a failure just now because of unset gh token, and this fixed it.

(the `$pr_number` thing was just a dumb dumb though)